### PR TITLE
2674: Add accessibility declaration to the footer

### PR DESCRIPTION
**Pull request by LeandraH**
### Short description

Adding the accessibility declaration to the footer for Integreat.

### Proposed changes

- Adding the accessibility declaration to the build config for Integreat. We probably need a separate one for Aschaffenburg at some point, and Malte doesn't need one.

### Resolved issues

Fixes: #2674 

---

### CODE REVIEW

**Review by ztefanie — APPROVED** [review]
Looks good :rocket: 

Tested on Integreat, with and without adding an englisch version, also tested malte. All works as expected :+1:

File: web/src/components/CityContentFooter.tsx [hunk 1]
```diff
@@ -34,10 +34,11 @@ type CityContentFooterProps = {
 }
 
 const CityContentFooter = ({ city, language, mode = 'normal' }: CityContentFooterProps): ReactElement => {
-  const { aboutUrls, privacyUrls } = buildConfig()
+  const { aboutUrls, privacyUrls, accessibilityUrls } = buildConfig()
   const { t } = useTranslation(['layout', 'settings'])
   const aboutUrl = aboutUrls[language] || aboutUrls.default
   const privacyUrl = privacyUrls[language] || privacyUrls.default
+  const accessibilityUrl: string | undefined = accessibilityUrls?.[language] ?? accessibilityUrls?.default
```
Comment (steffenkleinle) [inline web/src/components/CityContentFooter.tsx:41]: :wrench: Is the type actually necessary?
```suggestion
  const accessibilityUrl = accessibilityUrls?.[language] ?? accessibilityUrls?.default
```

File: web/src/components/GeneralFooter.tsx [hunk 2]
```diff
@@ -13,18 +13,20 @@ type GeneralFooterProps = {
 }
 
 const GeneralFooter = ({ language }: GeneralFooterProps): ReactElement => {
-  const { aboutUrls, privacyUrls } = buildConfig()
+  const { aboutUrls, privacyUrls, accessibilityUrls } = buildConfig()
   const { t } = useTranslation('layout')
 
   const aboutUrl = aboutUrls[language] || aboutUrls.default
   const privacyUrl = privacyUrls[language] || privacyUrls.default
+  const accessibilityUrl: string | undefined = accessibilityUrls?.[language] ?? accessibilityUrls?.default
```
Comment (steffenkleinle) [inline web/src/components/GeneralFooter.tsx:21]:
```suggestion
  const accessibilityUrl = accessibilityUrls?.[language] ?? accessibilityUrls?.default
```

**Review by steffenkleinle — APPROVED** [review]
Not tested, but looks good :+1:

File: web/src/components/CityContentFooter.tsx (hunk 1 above)
Comment (LeandraH) [reply web/src/components/CityContentFooter.tsx:41]: You're right, it's not necessary. The linter was complaining about the missing types but that must have been at a different version, it's not complaining now 🤷

**Pull request merged**